sema/rocreator/rocreator.yml

```
@@ -0,0 +1,53 @@
+  version: 1.2-DRAFT
+  profile:  !resolve {env.domain}/observatory-profile/latest
+
+
+license: https://creativecommons.org/cc-by/4.0/
+citation: "When using this work it should be attributed and refered to as this."
+
+
+
+agents:
+  https://orcid.org/....  # Marc Portier
+    roles: [ owner, contributor, contact ]
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 0 but found 4

(indentation)